pandas/core/indexes/base.py

```diff
@@ -1122,7 +1122,25 @@ def to_frame(self, index=True):
 
         Returns
         -------
-        DataFrame : a DataFrame containing the original Index data.
+        DataFrame
+            DataFrame containing the original Index data.
+
+        Examples
+        --------
+        >>> idx = pd.Index(['Ant', 'Bear', 'Cow'])
```

Comment:
Minor thing, but it'd probably look a bit nicer if you create the index with `name='animal'`. :) For the rest looks great to me.

Reply:
Well, I do think it is interesting when creating a new index:
```
>>> idx.to_frame(index = False)
  animal
0    Ant
1   Bear
2    Cow
```
but when using the same as index and column, it does not get 'nicer':

```
>>> idx.to_frame(index = True)
       animal
animal       
Ant       Ant
Bear     Bear
Cow       Cow
```